{"name": "test_update_task_invalid_id", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'status_code'", "trace": "self = <api.test_tasks.TestTasks object at 0x00000205714EEA40>\n\n    @allure.description(\"Обновление задачи с несуществующим ID\")\n    def test_update_task_invalid_id(self):\n        fake_task_id=\"nope-task-000\"\n        data = {\"name\": \"Should not work\"}\n        response= self.api.update_task(fake_task_id, data)\n>       assert response.status_code== 404, f\"Ожидали ошибку, получили {response.status_code}: {response.text}\"\nE       AttributeError: 'NoneType' object has no attribute 'status_code'\n\ntest_tasks.py:63: AttributeError"}, "description": "Обновление задачи с несуществующим ID", "start": 1749679228506, "stop": 1749679230134, "uuid": "f9f0c2e4-54e9-4436-b436-9ea5463c3c47", "historyId": "719d2b4da1dfef9b6adb89016e3ea409", "testCaseId": "719d2b4da1dfef9b6adb89016e3ea409", "fullName": "tests.api.test_tasks.TestTasks#test_update_task_invalid_id", "labels": [{"name": "feature", "value": "Clickup API: Tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "A1"}, {"name": "thread", "value": "30568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}